Makefile

```diff
@@ -0,0 +1,21 @@
+deploy: build
+	@git describe --tags > ./appversion
+	@heroku docker:release --app origami-registry-eu
+	@rm -f ./appversion
+
+build:
+	@if [[ "$$(docker-machine ls | grep dev)" == *"Stopped"* ]]; \
```

Comment:
I've tried this and can't seem to get Make to exit the task without throwing an error and finding it hard to find any good documentation on how to exit a task without make throwing an error. This wouldn't be a big issue but it's a dependency for the build task so I'm not sure how to exit the dependency and allow the other task to complete.